parser/internal/parserv2/parser.go

```diff
@@ -802,6 +806,7 @@ func (p *parser) parseFragments() ([]*ast.Fragment, error) {
 				return nil, err
 			}
 			fragments = append(fragments, fragment)
+			p.fragments[fragment.Name] = fragment
```

Comment:
we can drop the fragments slice now, and also remove it from the return?

Reply:
ah right 🤦‍♀️